integration-test/src/test/java/com/datastrato/gravitino/integration/test/catalog/hive/ProxyCatalogHiveIT.java

```diff
@@ -0,0 +1,267 @@
+        () -> anotherCatalog.asSchemas().createSchema(anotherIdent, comment, properties));
+  }
+
+  @Test
+  public void testOperateTable() throws Exception {
+    ColumnDTO[] columns = createColumns();
+    String schemaName = GravitinoITUtils.genRandomName(SCHEMA_PREFIX);
+    String tableName = GravitinoITUtils.genRandomName(TABLE_PREFIX);
+    String anotherTableName = GravitinoITUtils.genRandomName(TABLE_PREFIX);
+    NameIdentifier ident = NameIdentifier.of(METALAKE_NAME, CATALOG_NAME, schemaName);
+    NameIdentifier nameIdentifier =
+        NameIdentifier.of(METALAKE_NAME, CATALOG_NAME, schemaName, tableName);
+    NameIdentifier anotherNameIdentifier =
+        NameIdentifier.of(METALAKE_NAME, CATALOG_NAME, schemaName, anotherTableName);
+    Map<String, String> properties = Maps.newHashMap();
+    properties.put("key1", "val1");
+    properties.put("key2", "val2");
+    properties.put(
+        "location",
+        String.format(
+            "hdfs://%s:%d/user/hive/warehouse/%s.db",
+            containerSuite.getHiveContainer().getContainerIpAddress(),
+            HiveContainer.HDFS_DEFAULTFS_PORT,
+            schemaName.toLowerCase()));
+    String comment = "comment";
+    catalog.asSchemas().createSchema(ident, comment, properties);
+    Table createdTable =
+        catalog
+            .asTableCatalog()
+            .createTable(
+                nameIdentifier,
+                columns,
+                comment,
+                ImmutableMap.of(),
+                Partitioning.EMPTY_PARTITIONING);
+    String location = createdTable.properties().get("location");
+    Assertions.assertEquals(EXPECT_USER, hdfs.getFileStatus(new Path(location)).getOwner());
+    org.apache.hadoop.hive.metastore.api.Table hiveTab =
+        hiveClientPool.run(client -> client.getTable(schemaName, tableName));
+    Assertions.assertEquals(EXPECT_USER, hiveTab.getOwner());
+    Assertions.assertThrows(
+        RuntimeException.class,
+        () ->
+            anotherCatalog
+                .asTableCatalog()
+                .createTable(
+                    anotherNameIdentifier,
+                    columns,
+                    comment,
+                    ImmutableMap.of(),
+                    Partitioning.EMPTY_PARTITIONING));
+  }
+
+  private ColumnDTO[] createColumns() {
+    ColumnDTO col1 =
+        new ColumnDTO.Builder<>()
+            .withName("col1")
+            .withDataType(Types.ByteType.get())
+            .withComment("col_1_comment")
+            .build();
+    ColumnDTO col2 =
+        new ColumnDTO.Builder<>()
+            .withName("col2")
+            .withDataType(Types.DateType.get())
+            .withComment("col_2_comment")
+            .build();
+    ColumnDTO col3 =
+        new ColumnDTO.Builder<>()
+            .withName("col3")
+            .withDataType(Types.StringType.get())
+            .withComment("col_3_comment")
+            .build();
+    return new ColumnDTO[] {col1, col2, col3};
```

Comment:
Change all the ColumnDTO to use ColumnImpl API.

Reply:
Fixed.